### PR TITLE
Fix recipe remover and some recipe additions

### PR DESCRIPTION
**Pull request by kuba6000**
```
[09:55:04] [Client thread/INFO] [DREAMCRAFT/dreamcraft]: Removed 4254 recipes!
[09:55:04] [Client thread/INFO] [DREAMCRAFT/dreamcraft]: Recipes removal took 42 ms.
```
Fixes  https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/13492
Fixes https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/13494

### CODE REVIEW

**Comment by chochem** [comment]
not a code review. just loading it up and comparing it to the ticket. I still get these two:
image
image

**Comment by chochem** [comment]
thanks, those two look good now too

**Comment by wlhlm** [comment]
The wildcard meta value (32767), this that something specific to how GTNH Core Mod works, or is this a special value from RA? If the latter, I think we should add it as a proper constant in RA rather than have it as a variable in an interface here.

**Comment by kuba6000** [comment]
> The wildcard meta value (32767), this that something specific to how GTNH Core Mod works, or is this a special value from RA? If the latter, I think we should add it as a proper constant in RA rather than have it as a variable in an interface here.

It is just normal wildcard meta, also defined as a static constant variable in OreDictionary class I think. Nothing to do with RA

**Comment by OneEyeMaker** [comment]
Why don't you use the constant (`OreDictionary.WILDCARD_VALUE`) instead of "magic" number?

**Comment by kuba6000** [comment]
> Why don't you use the constant (`OreDictionary.WILDCARD_VALUE`) instead of "magic" number?

What do you mean? These recipe removals were ported straight from scripts and I just replaced all the magic wildcard numbers with a variable in IScriptLoader class named "wildcard" or did I miss something?

File: src/main/java/com/dreammaster/scripts/ScriptAdvancedSolarPanel.java [hunk 1]
```diff
@@ -31,65 +31,91 @@ public void loadRecipes() {
...
-                        getModItem(AdvancedSolarPanel.ID, "BlockAdvSolarPanel", 1, 0, missing),
-                        "craftingToolScrewdriver", "circuitAdvanced",
-                        getModItem(IndustrialCraft2.ID, "itemArmorNanoHelmet", 1, 32767, missing), "circuitAdvanced",
-                        "cableGt04Aluminium", getModItem(GregTech.ID, "gt.blockmachines", 1, 22, missing),
-                        "cableGt04Aluminium" });
+                "craftingToolWrench",
+                getModItem(AdvancedSolarPanel.ID, "BlockAdvSolarPanel", 1, 0, missing),
+                "craftingToolScrewdriver",
+                "circuitAdvanced",
+                getModItem(IndustrialCraft2.ID, "itemArmorNanoHelmet", 1, 32767, missing),
```
Comment (OneEyeMaker) [inline src/main/java/com/dreammaster/scripts/ScriptAdvancedSolarPanel.java:38]: This one and two more in this file.

File: src/main/java/com/dreammaster/scripts/ScriptAvaritiaAddons.java [hunk 2]
```diff
@@ -78,17 +89,21 @@ public void loadRecipes() {
...
+                getModItem(GregTech.ID, "gt.metaitem.01", 1, 32633),
+                "circuitElite",
+                getModItem(GregTech.ID, "gt.metaitem.01", 1, 32633));
 
         // clearing NBT of th
-        addShapelessCraftingRecipe(
+        addShapelessRecipe(
                 getModItem(AvaritiaAddons.ID, "ExtremeAutoCrafter", 1),
-                new Object[] { getModItem(AvaritiaAddons.ID, "ExtremeAutoCrafter", 1, 32767), "platePaper" });
+                getModItem(AvaritiaAddons.ID, "ExtremeAutoCrafter", 1, 32767),
```
Comment (OneEyeMaker) [inline src/main/java/com/dreammaster/scripts/ScriptAvaritiaAddons.java:105]: This one.

File: src/main/java/com/dreammaster/scripts/ScriptAdvancedSolarPanel.java (hunk 1 above)
Comment (kuba6000) [reply src/main/java/com/dreammaster/scripts/ScriptAdvancedSolarPanel.java:38]: oh ye i just replaced these in RecipeRemovals file

**Review by Dream-Master — APPROVED** [review]
works ingame so far i can see ingame

**Pull request merged**